<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>106</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_13">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/icon/menu_icon.svg</normaloff>:/icons/icon/menu_icon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>106</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>66</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>138</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="home_page">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>home page</string>
           </property>
          </widget>
          <widget class="QWidget" name="searchWidget" native="true">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>420</width>
             <height>161</height>
            </rect>
           </property>
           <widget class="QFrame" name="searchframe">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>400</width>
              <height>50</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">#searchframe{
border-radius: 25px;
border: 2px solid #71a5de;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="line_search_bar">
               <property name="styleSheet">
                <string notr="true">background: transparent;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_search">
               <property name="styleSheet">
                <string notr="true">border: none;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/icon/search_icon.svg</normaloff>:/icons/icon/search_icon.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_image_search">
               <property name="styleSheet">
                <string notr="true">border: none;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/icon/search_icon.svg</normaloff>:/icons/icon/search_icon.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QPushButton" name="btn_header">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>40</y>
              <width>161</width>
              <height>23</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string>LOGO</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="imageWidget" native="true">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>260</y>
             <width>421</width>
             <height>80</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="todayPrice_widget" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>360</y>
             <width>421</width>
             <height>321</height>
            </rect>
           </property>
           <widget class="QStackedWidget" name="todayPrice_stackedWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>401</width>
              <height>251</height>
             </rect>
            </property>
            <widget class="QWidget" name="home_widget_page_1">
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>101</width>
                <height>101</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/image/resource/image/gajami.jpg</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="label_fish_name_1">
              <property name="geometry">
               <rect>
                <x>130</x>
                <y>40</y>
                <width>67</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>가자미</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>81</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>오늘 거래가</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>40</y>
                <width>81</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>전일대비</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_11">
              <property name="geometry">
               <rect>
                <x>300</x>
                <y>40</y>
                <width>81</width>
                <height>17</height>
               </rect>
              </property>
              <property name="text">
               <string>전년대비</string>
              </property>
             </widget>
             <widget class="QWidget" name="today_price_widget" native="true">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>80</y>
                <width>81</width>
                <height>41</height>
               </rect>
              </property>
              <widget class="QLabel" name="label_today_price">
               <property name="geometry">
                <rect>
                 <x>9</x>
                 <y>9</y>
                 <width>45</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>25000</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_18">
               <property name="geometry">
                <rect>
                 <x>58</x>
                 <y>9</y>
                 <width>17</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>원</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="previous_day_price_widget" native="true">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>80</y>
                <width>81</width>
                <height>31</height>
               </rect>
              </property>
              <widget class="QLabel" name="label_previous_day_price">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>45</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>n</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_16">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>10</y>
                 <width>17</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>원</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="previous_year_price_widget" native="true">
              <property name="geometry">
               <rect>
                <x>300</x>
                <y>90</y>
                <width>91</width>
                <height>21</height>
               </rect>
              </property>
              <widget class="QLabel" name="label_previous_year_price">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>45</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>n</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_17">
               <property name="geometry">
                <rect>
                 <x>60</x>
                 <y>0</y>
                 <width>17</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>원</string>
               </property>
              </widget>
             </widget>
            </widget>
            <widget class="QWidget" name="page_3"/>
           </widget>
           <widget class="QPushButton" name="btn_today_price_stacked_widget_1">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>290</y>
              <width>20</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_today_price_stacked_widget_2">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>290</y>
              <width>20</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_today_price_stacked_widget_3">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>290</y>
              <width>20</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>3</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QToolButton" name="btn_today_price_stacked_widget_4">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>290</y>
              <width>20</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QToolButton" name="btn_today_price_stacked_widget_5">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>290</y>
              <width>20</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QToolButton" name="btn_today_price_stacked_widget_6">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>290</y>
              <width>20</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>6</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QToolButton" name="btn_today_price_stacked_widget_7">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>290</y>
              <width>20</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>7</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
           <widget class="QToolButton" name="btn_today_price_stacked_widget_8">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>290</y>
              <width>20</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>8</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="fish_list_page">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>list page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="safety_page">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>111</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>safety page</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="image_search_page">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>171</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>image search page</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color:#71a5de;
}

QPushButton{
	color: white;
	text-align: left;
	height: 30px;
	border: none;
	padding-left: 10px;
	padding-right: 20px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

QPushButton:checked{
	background-color: #ffffff;
	color:#71a5de;
	font-weight:bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_home_page_1">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>home</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/icon/home_white.svg</normaloff>
             <normalon>:/icons/icon/home_blue.svg</normalon>:/icons/icon/home_white.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_fish_list_page_1">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>list</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/icon/profile_white.png</normaloff>
             <normalon>:/icons/icon/profile_blue.png</normalon>:/icons/icon/profile_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_image_search_page_1">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>search</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/icon/apple_white.svg</normaloff>
             <normalon>:/icons/icon/apple.svg</normalon>:/icons/icon/apple_white.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_safety_page_1">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>safety</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/icon/baseball_ball_white.png</normaloff>
             <normalon>:/icons/icon/baseball_ball_blue.png</normalon>:/icons/icon/baseball_ball_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_13</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
